.github/workflows/spelling.yml

```diff
@@ -16,3 +16,5 @@ jobs:
           skip: ./src/control/vendor,./.git
           ignore_words_file: ci/codespell.ignores
           builtin: clear,rare,informal,names,en-GB_to_en-US
+      - name: Check DAOS logging macro use.
+        run: ./utils/cq/d_error_check.py --github src
```

Comment:
I will remove this prior to landing.

In theory this should probably go in it's own action however it's easy enough to add it to an existing one. That can be discussed when we're ready to enable this.

Reply:
Since we keep adding GHA, and want to add more, it probably makes sense to group similar checks.